### PR TITLE
doc: start a docs update

### PR DESCRIPTION
**Pull request by beckermr**
This PR starts to update the docs.

xref #89

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -18,3 +18,4 @@ repos:
         exclude: tests/GalSim/|tests/Coord/|tests/jax/galsim/
       - id: ruff-format
         exclude: tests/GalSim/|tests/Coord/|tests/jax/galsim/
+
```
Comment (beckermr) [inline .pre-commit-config.yaml:21]:
```suggestion
```

**Comment by beckermr** [comment]
@ismael-mendoza This PR is a start at updating the README. I think we'll want more docs before we can close issue #89.

**Comment by beckermr** [comment]
I'd like to merge this PR and tackle docstrings + other docs in another PR.

File: README.md [hunk 2]
```diff
@@ -6,23 +6,28 @@
...
-
-## Objective and design
-
-See [design document](https://docs.google.com/document/d/1NalCc_5dc3Z8F4q37y-RsJS_mr9gzvfyANb2PYUpsb4/edit?usp=sharing).
+## Objective and Design
 
 The goal of this library is to reimplement GalSim functionalities in pure JAX to allow for automatic differentiation, GPU acceleration, and batched computations.
 
-**Guiding Principles**:
+### Guiding Principles**
```
Comment (ismael-mendoza) [inline README.md:13]:
```suggestion
### Guiding Principles
```

**Review by ismael-mendoza — APPROVED** [review]
Nice addition! I also found one typo

**Pull request merged**